Index: risk/src/view/graphic/gui/fxml/SignInPage.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/risk/src/view/graphic/gui/fxml/SignInPage.fxml b/risk/src/view/graphic/gui/fxml/SignInPage.fxml
new file mode 100644
--- /dev/null	(date 1610207315907)
+++ b/risk/src/view/graphic/gui/fxml/SignInPage.fxml	(date 1610207315907)
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.PasswordField?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.effect.ColorAdjust?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.AnchorPane?>
+
+
+<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1" fx:controller="view.graphic.gui.fxml.SignInPage">
+   <children>
+      <ImageView fitHeight="600.0" fitWidth="800.0" pickOnBounds="true" preserveRatio="true">
+         <image>
+            <Image url="@../../../../../../ap99/page2.png" />
+         </image>
+      </ImageView>
+      <ImageView fitHeight="186.0" fitWidth="186.0" layoutX="309.0" layoutY="305.0" pickOnBounds="true" preserveRatio="true">
+         <image>
+            <Image url="@../../../../../../ap99/page-5.png" />
+         </image>
+      </ImageView>
+      <PasswordField layoutX="342.0" layoutY="386.0" prefHeight="25.0" prefWidth="123.0" promptText="password">
+         <effect>
+            <ColorAdjust brightness="-0.2" />
+         </effect>
+      </PasswordField>
+      <ImageView fitHeight="186.0" fitWidth="186.0" layoutX="306.0" layoutY="212.0" pickOnBounds="true" preserveRatio="true">
+         <image>
+            <Image url="@../../../../../../ap99/page-5.png" />
+         </image>
+      </ImageView>
+      <TextField layoutX="337.0" layoutY="292.0" prefHeight="25.0" prefWidth="126.0" promptText="user name">
+         <effect>
+            <ColorAdjust brightness="-0.2" />
+         </effect>
+      </TextField>
+      <ImageView fitHeight="150.0" fitWidth="200.0" layoutX="511.0" layoutY="416.0" pickOnBounds="true" preserveRatio="true">
+         <image>
+            <Image url="@../../../../../../ap99/page-3.png" />
+         </image>
+      </ImageView>
+   </children>
+</AnchorPane>
Index: risk/src/view/graphic/gui/fxml/SignUpPage.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/risk/src/view/graphic/gui/fxml/SignUpPage.fxml b/risk/src/view/graphic/gui/fxml/SignUpPage.fxml
new file mode 100644
--- /dev/null	(date 1610207598533)
+++ b/risk/src/view/graphic/gui/fxml/SignUpPage.fxml	(date 1610207598533)
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<AnchorPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:controller="view.graphic.gui.fxml.SignUpPage"
+            prefHeight="400.0" prefWidth="600.0">
+
+</AnchorPane>
Index: risk/src/view/graphic/gui/fxml/style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/risk/src/view/graphic/gui/fxml/style.css b/risk/src/view/graphic/gui/fxml/style.css
new file mode 100644
--- /dev/null	(date 1610213337924)
+++ b/risk/src/view/graphic/gui/fxml/style.css	(date 1610213337924)
@@ -0,0 +1,3 @@
+.style{
+    -fx
+}
\ No newline at end of file
